src/custom/utils/operator/errors/OperatorError.ts

```diff
@@ -30,12 +33,25 @@ export enum ApiErrorCodeDetails {
   UnsupportedToken = 'One of the tokens you are trading is unsupported. Please read the FAQ for more info.',
   WrongOwner = "The signature is invalid.\n\nIt's likely that the signing method provided by your wallet doesn't comply with the standards required by CowSwap.\n\nCheck whether your Wallet app supports off-chain signing (EIP-712 or ETHSIGN).",
   NotFound = 'Token pair selected has insufficient liquidity',
-  FeeExceedsFrom = 'Fee amount for selected pair exceeds "from" amount',
   OrderNotFound = 'The order you are trying to cancel does not exist',
+  UNHANDLED_GET_ERROR = 'Order fetch failed. This may be due to a server or network connectivity issue. Please try again later.',
   UNHANDLED_CREATE_ERROR = 'The order was not accepted by the network',
   UNHANDLED_DELETE_ERROR = 'The order cancellation was not accepted by the network'
 }
 
+function _mapActionToErrorDetail(action?: ApiActionType) {
+  switch (action) {
+    case 'get':
+      return ApiErrorCodeDetails.UNHANDLED_GET_ERROR
+    case 'create':
+      return ApiErrorCodeDetails.UNHANDLED_CREATE_ERROR
+    // default and last case..
+    case 'delete':
+    default:
+      return ApiErrorCodeDetails.UNHANDLED_DELETE_ERROR
```

Comment:
You don't need a default if all cases are covered.
But since it could be `undefined`, I wouldn't use the same error as `delete`.

What about 'Something failed, try again later bla bla...'

Reply:
ah true!